Change quickstarts to use new secrets format (#137)
spiceai/quickstarts: catalogs/spiceai/README.md

Sign up for a Spice.ai Cloud Platform account at https://spice.ai.

## Step 2. Create a new directory and initialize a Spicepod

```bash
mkdir spice-catalog-demo
cd spice-catalog-demo
spice init
```

## Step 3. Login to the Spice.ai Cloud Platform with `spice login`

> Using the Spice CLI, login to the Spice.ai Cloud Platform. A window will open in your browser to authenticate. Ensure you are in the `spice-catalog-demo` directory.

Reviewer: ```suggestion
Working in the `spice-catalog-demo` directory, use the Spice CLI to login to the Spice.ai Cloud Platform. A browser window will open to authenticate when executing the `spice login` command.
```

```bash
spice login
```

> The API Key and token required to communicate with the Spice.ai Cloud Platform will be stored in a `.env` file in the directory where `spice login` was run, which will be `spice-catalog-demo` in this case. The Spice runtime automatically reads the `.env` file in the current working directory to load environment variables.

Reviewer: ```suggestion
After successfully authenticating, the Spice.ai Cloud Platform API Key and Token will be stored in the `spice-catalog-demo` working directory `.env` file. The Spice runtime reads environment variables set in the local working `.env` file.
```
